id-repository/id-repository-credentials-feeder/pom.xml

```diff
@@ -20,8 +20,8 @@
 		<id.repository.core.version>${parent.version}</id.repository.core.version>
 
```

Comment:
changed the pom version 1.2.1-SNAPSHOT to 1.2.1-java21-SNAPSHOT

Reply:
Changed version to java21 snapshot.